src/Simplex/Chat/Help.hs

```diff
@@ -44,32 +37,47 @@ chatWelcome :: User -> Onboarding -> [StyledString]
 chatWelcome user Onboarding {contactsCount, createdGroups, membersCount, filesSentCount, addressCount} =
   map
     styleMarkdown
-    [ blue "                             __   __",
-      cyan "  ___ ___ __  __ ___ _    ___" <> blue "\\ \\ / /" <> yellow " ___ _  _   _ _____",
-      cyan " / __|_ _|  \\/  | _ \\ |  | __ " <> blue "\\ V /" <> yellow " / __| || | /_\\_   _|",
-      cyan " \\__ \\| || |\\/| |  _/ |__| _|" <> blue " / . \\" <> yellow "| (__| __ |/ _ \\| |",
-      cyan " |___/___|_|  |_|_| |____|___" <> blue "/_/ \\_\\" <> yellow "\\___|_||_/_/ \\_\\_|",
+    [ "                             __   __",
+      "  ___ ___ __  __ ___ _    ___" <> "\\ \\ / /" <> " ___ _  _   _ _____",
+      " / __|_ _|  \\/  | _ \\ |  | __ " <> "\\ V /" <> " / __| || | /_\\_   _|",
+      " \\__ \\| || |\\/| |  _/ |__| _|" <> " / . \\" <> "| (__| __ |/ _ \\| |",
+      " |___/___|_|  |_|_| |____|___" <> "/_/ \\_\\" <> "\\___|_||_/_/ \\_\\_|",
       "",
       "Welcome " <> green userName <> "!",
       "Thank you for installing SimpleX Chat!",
       "",
-      "To try out how it works:",
+      "We have created several groups that you can join to play with SimpleX Chat:",
+      highlight "#simplex" <> " (SimpleX Engineers 💻) - technical questions about running or contributing to SimpleX Chat",
+      highlight "#hacks" <> " (Ethical Hacking 🔓) - chatting about privacy, security, announced vulnerabilities etc.",
+      highlight "#music" <> " (Music 🎸) - favorite music of our team and users",
+      highlight "#rand" <> " (Random 😇) - anything interesting, just keep it decent and friendly please :)",
+      "",
+      "Connect to our groups admin to be added to these groups - " <> highlight "/admin",
+      "",
+      "To continue:",
       "[" <> check (contactsCount >= 2) <> "] connect with 2 friends - " <> highlight "/help" <> " for instructions",
-      "[" <> check (createdGroups >= 1 && membersCount >= 2) <> "] create a group with them - " <> highlight "/group #friends",
-      "[" <> check (filesSentCount >= 1) <> "] send your photo, e.g. to the group - " <> highlight "/file #friends ./photo.jpg",
-      "[" <> check (addressCount >= 1) <> "] create your chat " <> highlight "/address" <> " and share it with your friends",
+      "[" <> check (createdGroups >= 1 && membersCount >= 2) <> "] create a " <> highlight "/group" <> " with them - " <> highlight "/g #friends",
+      "[" <> check (filesSentCount >= 1) <> "] send " <> highlight "/file" <> ", e.g. your photo, to the group - " <> highlight "/f #friends ./photo.jpg",
+      "[" <> check (addressCount >= 1) <> "] create your chat " <> highlight "/address" <> " and share it with your friends - " <> highlight "/ad",
```

Comment:
It only occurred to me that it wouldn't be obvious to new users that address is not necessary, which makes question the "no id" feature. It does go last so it makes it more obvious, but maybe we should add some explicit explanation right away